src/Workspaces/CSharp/Portable/Utilities/FormattingRangeHelper.cs

```diff
@@ -360,7 +360,11 @@ private static bool IsSpecialContainingNode(SyntaxNode node)
                 node.Kind() == SyntaxKind.FinallyClause ||
                 node.Kind() == SyntaxKind.LabeledStatement ||
                 node.Kind() == SyntaxKind.LockStatement ||
-                node.Kind() == SyntaxKind.FixedStatement;
```

Comment:
not a problem witht his PR, but i have no idea what "special-containing-node" means.   What does it mean to be "special"?  Why are these nodes "special"? How do i judge if the added nodes are correct to have here, and that we're not missing other nodes that should go here?

@heejaechang can you clarify the intent of this method?

Reply:
it is basically list of nodes that require different behavior than normal one. special cases basically. usually means one larger scope than other cases. but not sure it is still that since it has been years since I last touched this.

looks like IsSpecialContainingNode should only be used in GetTopContainingNode and nothing else.